### PR TITLE
chore: Switch to gtm_barebones

### PR DESCRIPTION
**Pull request by attiks**
Replace google_tag with gtm_barebones

Refs: [OPS-10039](https://humanitarian.atlassian.net/browse/OPS-10039)

[OPS-10039]: https://humanitarian.atlassian.net/browse/OPS-10039?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/monitoring.sensor_config.ocha_google_tag.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
-uuid: 387db73c-d76b-4cf8-aec2-28e9c8531804
+uuid: null
```
Comment (lazysoundsystem) [inline config/monitoring.sensor_config.ocha_google_tag.yml:1]: Think this config should be removed

Comment (attiks) [reply config/monitoring.sensor_config.ocha_google_tag.yml:1]: It will be created again by ocha_monitoring

Comment (attiks) [reply config/monitoring.sensor_config.ocha_google_tag.yml:1]: See https://github.com/UN-OCHA/ocha_monitoring/pull/4

**Comment by attiks** [comment]
@lazysoundsystem I cannot merge it

Merge attempt failed
Head branch is out of date. Review and try the merge again.

Can you try?

**Comment by attiks** [comment]
I saw something similar, when it was trying to decide if it could be merged or not

**Pull request merged**